src/imitation/algorithms/preference_comparisons.py

```diff
@@ -142,8 +143,19 @@ def __init__(
         # will set self.logger, which also sets the logger for the algorithm
         super().__init__(custom_logger)
         if isinstance(reward_fn, reward_nets.RewardNet):
-            reward_fn = reward_fn.predict
+            if isinstance(reward_fn, reward_nets.NormalizedRewardNet):
```

Comment:
Why do we need to check this? If it's not a `NormalizedRewardNet`, it's harmless to have the partial application with `update_norm_stats=False`, it'll just be a no-op.

Reply:
Ah I see you only added `update_norm_stats` to `NormalizedRewardNet.predict_processed` not the base version. That makes sense, it is as written quite specific to normalization. But maybe we can just make the keyword argument more generic, e.g. `update_stats` or `is_rl`?

It's generally a bit error prone to have code depend on the types. If we add some new wrapper that implements its own `predict_processed`, it won't be obvious that we need to update preference comparisons to handle it.